src/index.jl

```diff
@@ -586,7 +531,7 @@ function Base.show(io::IO, i::Index)
   end
 end
 
-Base.show(io::IO, iv::IndexVal) = print(io, ind(iv), "=>$(val(iv))")
+Base.show(io::IO, iv::Pair{<:Index}) = print(io, ind(iv), "=>$(val(iv))")
```

Comment:
My guess is that we don't need this anymore, since Julia should already print it like this.